### DISCUSSION TITLE
Unable to Host the Website

### DISCUSSION THREAD

**Discussion by kamyashinde** [community/community#23141] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, I am fairly new to Github. I have hosted a few website but the one I have problems with is a huge on. I am trying to host a website, but it shows the readme file after publishing it. It would be really helpful if someone could tell me what is wrong with my files as soon as possible.<br>
Thankyou very much!</p>
<p><a href="https://github.com/kamyashinde/old_Bharatanatyam" rel="noopener nofollow ugc">https://github.com/kamyashinde/old_Bharatanatyam</a></p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/kamyashinde">@kamyashinde</a>, welcome to the GitHub Support Community! Pages serves a file called <code>index.html</code>, <code>index.md</code>, or <code>README.md</code> by default, in that order. As you don’t have a file called <code>index.html</code> Pages is serving your README file.</p>
<p>You can see your site online by visiting the page directly:</p>
<p><a href="https://kamyashinde.github.io/old_Bharatanatyam/new-home-page.html" class="onebox" target="_blank" rel="noopener">https://kamyashinde.github.io/old_Bharatanatyam/new-home-page.html</a></p>
<p>If you’d like to serve this by default then you’ll need to rename it from <code>new-home-page.html</code> to <code>index.html</code>.</p>

**Comment by kamyashinde** [comment]
<p>That’s so true. Thank you very much!</p>
